### PR TITLE
Upgrade: 4.1.1

### PR DESCRIPTION
**Pull request by Mcastres**
Just a quick PR to:

- Update on 4.1.1
- Add SEO plugin
- Add more data in the SEO component

### CODE REVIEW

File: api/package.json [hunk 1]
```diff
@@ -27,11 +28,11 @@
     "name": "A Strapi developer"
   },
   "strapi": {
-    "uuid": "092c44de-692e-4d21-acfa-ed40a3fe01a9"
+    "uuid": "FOODADVISOR-LOCAL-092c44de-692e-4d21-acfa-ed40a3fe01a9"
```
Comment (ivanThePleasant) [inline api/package.json:31]: Why the prefix is LOCAL? Is this for local use only?

File: api/config/server.js [hunk 2]
```diff
@@ -1,4 +1,7 @@
 module.exports = ({ env }) => ({
   host: env('HOST', '0.0.0.0'),
   port: env.int('PORT', 1337),
+  app: {
```
Comment (ivanThePleasant) [inline api/config/server.js:4]: Not a critique, I'm just not getting what this is doing for some reason 😅

Comment (Mcastres) [reply api/config/server.js:4]: This is needed for the [migration of 4.0.0 to 4.0.6](https://docs.strapi.io/developer-docs/latest/update-migration-guides/migration-guides/v4/migration-guide-4.0.x-to4.0.6.html) apparently

**Review by ivanThePleasant — APPROVED** [review]
Thanks for clarifying my questions. All good.

**Pull request merged**